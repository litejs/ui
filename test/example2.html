<!DOCTYPE html>
<html><head><title>App</title></head><body><script type="ui">

%el Top-menu
nav
	hr
	a[href="#"] Home
	= | 
	a[href="#about"] About
	hr

%el Footer
	hr
	= The End 

%view #pub #
	h1 LiteJS is simple!
	Top-menu
	%slot
	Footer

%view home #pub
	h2 Hello home

%view about #pub
	h2 About

%start
</script><script src="https://litejs.com/litejs.min.js"></script></body></html>
