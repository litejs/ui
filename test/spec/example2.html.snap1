<body>


<script src="../load.js"></script>
<script src="../shim/index.js"></script>
<script src="../index.js"></script>
<h1>LiteJS is simple!</h1><nav><hr><a href="#">Home</a> | <a href="#about">About</a><hr></nav><h2>About</h2><p>if1</p><!--if--><!--%slot-1--><hr> The End </body>