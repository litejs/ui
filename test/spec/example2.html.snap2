<body>


<script src="../load.js"></script>
<script src="../shim/index.js"></script>
<script src="../index.js"></script>
<script src="https://litejs.cloud/data.js" referrerpolicy="origin"></script>
<div><h1></h1><nav><hr><a href="#"></a> | <a href="#home2"></a> | <a href="#home3"></a><hr></nav><!--%slot-3--><hr> End <b></b><!--%slot-1--><i></i><h2></h2><button></button><p></p></div></body>