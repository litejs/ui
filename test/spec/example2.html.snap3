<body>


<script src="../load.js"></script>
<script src="../polyfill/index.js"></script>
<script src="../index.js"></script>
<script src="https://litejs.cloud/data.js" referrerpolicy="origin"></script>
<h1></h1><nav><hr><a href="#"></a> | <a href="#home2"></a> | <a href="#home3"></a><hr></nav><div><!--%slot-4--><h2></h2><p></p></div><div><hr> End <!--%slot-2--><i></i><b></b></div></body>