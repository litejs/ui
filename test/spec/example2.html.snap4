<body>


<script src="../load.js"></script>
<script src="../polyfill/index.js"></script>
<script src="../index.js"></script>
<script src="https://litejs.cloud/data.js" referrerpolicy="origin"></script>
<div><h1></h1><nav><hr><a href="#"></a> | <a href="#home2"></a> | <a href="#home3"></a><hr></nav><div><!--%slot-5--></div><hr> End <b></b><!--%slot-1--><i></i><h2></h2><p></p></div></body>