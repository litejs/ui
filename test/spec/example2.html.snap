<body>


<script src="../load.js"></script>
<script src="../polyfill/index.js"></script>
<script src="../index.js"></script>
<h1></h1><nav><hr><a href="#"></a> | <a href="#about"></a><hr></nav><h2></h2><!--%slot-1--><hr> The End </body>